src/main/java/seedu/address/logic/commands/module/DeleteModuleCommand.java

```diff
@@ -21,18 +21,28 @@
 
     public static final String MESSAGE_USAGE = COMMAND_WORD
             + ": Deletes the module identified by the index number used in the displayed module list.\n"
+            + "The module must not have any associated upcoming meetings.\n"
```

Comment:
```suggestion
            + "The module must not have any associated meetings.\n"
```

I think the current implementation doesn't allow deletion even if the associated meetings have already expired, hence we should probably remove that part.

Reply:
Okay, but I'll change this to `associated upcoming or expired meetings` for clarity on the users end, since it may not be intuitive that users will have to delete expired meetings first. What do you think?

Reply:
I think that's fine 👍